app/services/moves/updater.rb

```diff
@@ -43,11 +47,14 @@ def attributes
 
       attributes[:documents] = Document.where(id: document_ids) unless document_attributes.nil?
 
+      # TODO: to be removed once move profile migration complete
       unless person_attributes.nil?
         person = Person.find_by(id: person_attributes.dig(:data, :id))
         attributes[:profile] = person&.latest_profile
```

Comment:
maybe this should not be `person&.latest_profile` if `profile_id` is also present in the attributes ?

Reply:
I wanted to avoid checking both to avoid adding more complexity and tests for something going away soon, but happy to add it if its a risk

Reply:
👍

Reply:
spoke in person: updated this to guard against suppliers don't accidentally pass both in